src/widget/wmainmenubar.cpp

```diff
@@ -0,0 +1,692 @@
+#include "widget/wmainmenubar.h"
+
+#include <QDesktopServices>
+
+#include "controlobjectslave.h"
+#include "defs_urls.h"
+#include "mixer/playermanager.h"
+#include "util/cmdlineargs.h"
+#include "util/experiment.h"
+#include "vinylcontrol/defs_vinylcontrol.h"
+
+namespace {
+
+QString buildWhatsThis(const QString& title, const QString& text) {
```

Comment:
I happen to be the original author of this code I'm copying from MixxxMainWindow so I can tell you that it is used to build the string we set on the WhatsThis of our menu bar actions. :)

http://doc.qt.io/qt-4.8/qaction.html#whatsThis-prop
http://doc.qt.io/qt-4.8/qwhatsthis.html